packages/marble-button/src/Button.soy

```diff
@@ -77,18 +78,24 @@
     {if not $href}
       type="{$type}"
     {/if}
+
+    {if $tabIndex}
+      tabIndex="{$tabIndex}"
+    {/if}
   {/let}
 
   {if $href}
    <a {$attributes}>
   {else}
    <button {$attributes}>
   {/if}
+  <div class="btn-content-wrapper">
```

Comment:
why do you need this guy?

Reply:
Very true - just rethought it and we don't need this